webserver/views/api/v1/test/test_core.py

```diff
@@ -274,14 +274,134 @@ def test_get_bulk_hl_absent_mbid(self, load_many_high_level):
                       ("405a5ff4-7ee2-436b-95c1-90ce8a83b359", 2)]
         load_many_high_level.assert_called_with(recordings)
 
-    def test_get_bulk_hl_more_than_200(self):
+    def test_get_bulk_hl_more_than_25(self):
         # Create many random uuids, because of parameter deduplication
         manyids = [str(uuid.uuid4()) for i in range(26)]
         limit_exceed_url = ";".join(manyids)
         resp = self.client.get('api/v1/high-level?recording_ids=' + limit_exceed_url)
         self.assert400(resp)
         self.assertEqual('More than 25 recordings not allowed per request', resp.json['message'])
 
+    def test_get_bulk_select_features_no_params(self):
+        # No parameters in bulk lookup results in an error
+        resp = self.client.get('/api/v1/low-level/select')
+        self.assertEqual(400, resp.status_code)
+
+        # There are two params, but the recordings parameter is checked first
+        expected_result = {"message": "Missing `recording_ids` parameter"}
+        self.assertEqual(expected_result, resp.json)
+
+        # Only recordings parameter missing
+        features = "lowlevel.average_loudness;rhythm.onset_rate"
+        resp = self.client.get('/api/v1/low-level/select?features=%s' % features)
+        self.assertEqual(400, resp.status_code)
+
+        expected_result = {"message": "Missing `recording_ids` parameter"}
+        self.assertEqual(expected_result, resp.json)
+
+        # Only features parameter missing
+        recordings = self.test_recording1_mbid + ';' + self.test_recording2_mbid
+        resp = self.client.get('/api/v1/low-level/select?recording_ids=%s' % recordings)
+        self.assertEqual(400, resp.status_code)
+
+        expected_result = {"message": "Missing `features` parameter"}
+        self.assertEqual(expected_result, resp.json)
+
+    # Once PR #349 adds conversion to lowercase in _parse_bulk_params, this skip should be removed
+    @unittest.skip('Conversion to lowercase in _parse_bulk_params not yet added via PR #349.')
```

Comment:
@alastair this now uses unitttest.skip until #349 is merged - sorry to forget about adding this until now.